index.html

```diff
@@ -1,14 +1,17 @@
 <!doctype html>
-
 <html lang="en">
 <head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link rel="stylesheet"  type="text/css" href="scss/application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <p class="book-prop">Título del libro: <span class="book-prop-value">(género)</span></p>
```

Comment:
Instead of using a `span` tag inside the `p` tag, it's better to have an `h` tag with a `span` one as siblings and a parent div with `display: flex`.